### ISSUE TITLE
Format/Code-free elements

### ISSUE THREAD

**Issue by yelch** [tlovett1/custom-contact-forms#209]
I have an enhancement request: 
Would it be possible to add format/code-free elements: When adding a html or other type element, it always be embraced by <div></div>. For example I've tried to add a <span> element into a html-element:

```html
<div class="field skip-field html-2 html field-101 "> <span id="Section1"> </div>
```

Could you add a possibility to add a completely format/code-free elements?
My intention is to be able to create sections to hide/unhide elements

**Comment by ArmandoValencia** [comment]
I agree with this, i would like to use the free-code html block to add left and right floating columns in my form, but html blocks gets embarcad.
